### ISSUE TITLE
Could not get JavaFX Packager Tools

### ISSUE THREAD

**Issue by antoine-briand** [schulzp/uni-myo-recorder#1]
Hello 👋 

I'm trying to run your project (gradle run). I know it's a four years old project, but do you have any idea on solving this issue : 

`> Could not get JavaFX Packager Tools, please set one of [jfxrtDir in Gradle Properties, JFXRT_HOME in System Environment, JAVA_HOME in System Environment, java.home in JVM properties]`

I've tried with Amazon Corretto, Open JDK, Adoptopenjdk and Oracle JDK from version 8 to 12 but I always get this error.

Any help would be appreciated 👼

**Comment by schulzp** [comment]
Hi @antoine-briand! I'll look into it.

**Comment by schulzp** [comment]
Hi @antoine-briand! 

I upgraded to gradle 4.8.1, fixed a wrong `mainClass` FQCN, and updated the [README](https://github.com/schulzp/uni-myo-recorder/blob/master/README.md). Please let me know if you can get it to work now, too. 

*TL;DR* Make sure your JDK comes with a JavaFX runtime (`jfxrt.jar`) and it's available under the current `org.gradle.java.home`.
